crates/edr_solidity_tests/tests/it/invariant.rs

```diff
@@ -778,3 +790,177 @@ async fn test_invariant_scrape_values() {
         ]),
     );
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_invariant_roll_fork_handler() {
+    let path_pattern = ".*fuzz/invariant/common/InvariantRollFork.t.sol";
+
+    let _runner = TEST_DATA_DEFAULT
+        .runner_with_invariant_config_and_seed(
+            U256::from(119u32),
+            TestInvariantConfig {
+                runs: 2,
+                depth: 4,
+                ..TestInvariantConfig::default()
+            },
+        )
+        .await;
+
+    // let results = runner
+    //     .test_collect(SolidityTestFilter::new(
+    //         "invariant_fork_handler_block",
+    //         "InvariantRollForkBlockTest",
+    //         path_pattern,
+    //     ))
+    //     .await;
+    //
+    // assert_multiple(
+    //     &results,
+    //     BTreeMap::from([(
+    //         "default/fuzz/invariant/common/InvariantRollFork.t.sol:
+    // InvariantRollForkBlockTest",         vec![(
+    //             "invariant_fork_handler_block()",
+    //             false,
+    //             Some("revert: too many blocks mined".into()),
+    //             None,
+    //             None,
+    //         )],
+    //     )]),
+    // );
```

Comment:
Was this directly copied when applying patches or is this leftover from fixing something locally?

Reply:
Good catch thanks! This was commented out while debugging something, but I added it back now in 27c8af376dd071e161166aad36686a818e652b40